honeybee_radiance/lightpath.py

```diff
@@ -0,0 +1,83 @@
+"""Utilities to determine the path of light taken through interior spaces of a model."""
+from honeybee.boundarycondition import Surface
+
+
+def light_path_from_room(model, room_identifier):
```

Comment:
This is a great functionality and is only available thanks to the well structured model. Really like it. I can imagine similar methods can be useful for automatically setting up airflow networks or visualize the path for air.

One comment about fixing the name for `static_apertures`. Do you think it will be possible to change the default name here. If we want to keep it fixed which is fine to me then I should not include this in the folder `cfg` file which means user can never change it.

Reply:
Thank you!  Yes, there will be similar methods for doing this with Airflow Networks and for the thermal comfort maps, where we usually want to export models for view factor calculation in groups of rooms that are inter-connected by air walls.  At a certain point, I should probably write this method to be recursive (right now it only calculates the path up to one Room away). I'll add a `TODO` for this in the code.

Hm. Having `static_apertures` as a fixed name would probably make our lives easier but I don't know what the users will think about it. I can expose it as an option on this method for now and I trust your judgement about what to do with the `cfg` file.